requirements.unittest.txt

```diff
@@ -0,0 +1,17 @@
+#Install the agent
+-e .
+
+# app
+git+https://github.com/openstack/neutron@9.1.0
```

Comment:
We should also use stable/liberty for the version of neutron.  This version points to Newton.